exec/java-exec/src/main/java/org/apache/drill/exec/store/ActionOnFile.java

```diff
@@ -0,0 +1,68 @@
+package org.apache.drill.exec.store;
+
+import org.apache.drill.common.config.CommonConstants;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The action on the {@link CommonConstants#STORAGE_PLUGINS_OVERRIDE_CONF} file being performed after it's use
+ */
+public enum ActionOnFile {
+
+  NONE {
+    @Override
+    void action(URL url) {
+      // nothing to do
+    }
+  },
+  RENAME {
+    @Override
+    void action(URL url) {
+      File pluginsOverrideFile = new File(url.getPath());
+      String oldName = CommonConstants.STORAGE_PLUGINS_OVERRIDE_CONF;
```

Comment:
No need to use constant, file name can be taken from URL and thus `ActionOnFile` can be used for other files, not only for storage plugin override conf.

Reply:
Done